### ISSUE TITLE
Undocumented Physics component 

### ISSUE THREAD

**Issue by kevinsimper** [craftyjs/Crafty#577]
There is a very small component in `2D.js` called Physics. It is not used anywhere, and it seems that it maybe was an early version of the Gravity component.

2D.js line 1012:

```
Crafty.c("Physics", {
    _gravity: 0.4,
    _friction: 0.2,
    _bounce: 0.5,

    gravity: function (gravity) {
        this._gravity = gravity;
    }
});
```

**Comment by potomak** [comment]
In my opinion there're a lot of useless components that make this library almost unmaintainable and this is a _dramatic_ example.

**Comment by starwed** [comment]
Yeah, this should probably just be excised from the code base.

**Comment by kevinsimper** [comment]
Fixed in #585
